packages/jellyfish-transaction-builder/src/txn/txn_builder.ts

```diff
@@ -9,7 +9,7 @@ import {
 import { BigNumber } from 'bignumber.js'
 import { EllipticPairProvider, FeeRateProvider, Prevout, PrevoutProvider } from '../provider'
 import { calculateFeeP2WPKH } from './txn_fee'
-import { OP_DEFI_TX } from '@defichain/jellyfish-transaction/dist/script/defi'
+import { OP_DEFI_TX } from '@defichain/jellyfish-transaction/src/script/defi'
```

Comment:
This is quite tricky, there are 3 ways around this.

#### Only use imports that are exported in `index.ts`
> Simple

#### Alternatively we could add another resolver: (3 steps in total)
> Complex

```json
"@defichain/jellyfish-transaction": ["jellyfish-transaction/src"],
"@defichain/jellyfish-transaction/*": ["jellyfish-transaction/src/*"],
```
Then we could do this
```ts
import { OP_DEFI_TX } from '@defichain/jellyfish-transaction/script/defi'
```
After building, we need to move all dist into root directory `packages/jellyfish-transaction/` instead of `packages/jellyfish-transaction/dist`

#### Lastly in TypeScript 4.3

> TypeScript 4.3

We could declare custom mapping, but it's not stable atm. It's super new a feature that is <1 month old.

- https://github.com/microsoft/TypeScript/issues/42762
- https://github.com/microsoft/TypeScript/issues/33079

We need this merged in first: https://github.com/DeFiCh/jellyfish/pull/321

Reply:
I don't mind using bleeding-edge features. I am in favour of using the latest features.